db/seed-helpers.js

```diff
@@ -115,13 +117,13 @@ var createRandomStock = (ticker, name, analystBuy, platformOwners, latestPrice,
   for (var i = 0; i < dataPoints ; i++) {
     var lastPrice = newStock.last1dPrices[newStock.last1dPrices.length - 1].price;
     var newDateTime = new Date(newStock.last1dPrices[newStock.last1dPrices.length - 1].datetime);
-    if (newDateTime.getHours() > 16) {
-      newDateTime.setHours(16);
-      newDateTime.setMinutes(0);
+    if (newDateTime.getHours() > 18) {
```

Comment:
there's a fun idea that some people like to use, which is "always use `<`, and never use `>`"